projects/germany/pelias.json

```diff
@@ -0,0 +1,59 @@
+{
+  "logger": {
+    "level": "debug",
+    "timestamp": false
+  },
+  "esclient": {
+    "apiVersion": "7.5",
+    "hosts": [
+      { "host": "elasticsearch" }
+    ]
+  },
+  "elasticsearch": {
+    "settings": {
+      "index": {
+        "refresh_interval": "10s",
+        "number_of_replicas": "0",
+        "number_of_shards": "1"
+      }
+    }
+  },
+  "api": {
+    "services": {
+      "pip": { "url": "http://pip:4200" },
+      "libpostal": { "url": "http://libpostal:4400" },
+      "placeholder": { "url": "http://placeholder:4100" },
+      "interpolation": { "url": "http://interpolation:4300" }
+    }
+  },
+  "imports": {
+    "geonames": {
+      "datapath": "/data/geonames",
+      "countryCode": "DE"
+    },
+    "openstreetmap": {
+      "download": [
+        { "sourceURL": "http://download.geofabrik.de/europe/germany-latest.osm.pbf" }
+      ],
+      "leveldbpath": "/tmp",
+      "datapath": "/data/openstreetmap",
+      "import": [{
+        "filename": "germany-latest.osm.pbf"
+      }]
+    },
+    "openaddresses": {
+      "datapath": "/data/openaddresses",
+      "files": ["de/berlin.csv", "de/he/city_of_frankfurtammain.csv", "de/he/city_of_kassel.csv", "de/hh/statewide.csv", "de/ni/bentheim.csv", "de/nw/city_of_cologne.csv", "de/nw/city_of_wesel.csv", "de/nw/city_of_wuppertal.csv", "de/nw/statewide.csv", "de/sn/statewide.csv", "de/th/statewide.csv"]
+    },
+    "polyline": {
+      "datapath": "/data/polylines",
+      "files": [ "extract.0sv" ]
+    },
+    "whosonfirst": {
+      "datapath": "/data/whosonfirst",
+      "countryCode": "DE",
+      "importPostalcodes": true,
```

Comment:
Heya, sorry I just double-checked this, the [whosonfirst-data-postalcode-de](https://github.com/whosonfirst-data/whosonfirst-data-postalcode-de) repo which is the source of these postalcodes doesn't actually contain any useful info.

I had a look through the data and all the records are marked non-current and all the lat/lon values are 0,0.

Can we please set `importPostalcodes` to `false`?

Having it as `false` will prevent users downloading that file from our CDN for no reason, saving everyone bandwidth and time 😄

Reply:
Ah, of course - should've spotted that myself. That's a shame - I'll set it to false and resubmit, then see if I can get some updated data there to update it - seems like it should be doable.

Reply:
Great thanks, once that's done we can merge :shipit: